### PR TITLE
fix(#1774): fix & update gitbook links

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- update gitbook links

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1774)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: govtool/frontend/src/components/organisms/DashboardCards/DRepDashboardCard.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ export const DRepDashboardCard = ({
               dataTestId: "register-learn-more-button",
               onClick: () =>
                 openInNewTab(
-                  "https://docs.gov.tools/how-to-use-the-govtool/using-govtool/dreps/retire-as-a-drep",
+                  "https://docs.gov.tools/using-govtool/govtool-functions/dreps/register-as-a-drep",
                 ),
             }
           : learnMoreButton,
```
Comment (jdyczka) [inline govtool/frontend/src/components/organisms/DashboardCards/DRepDashboardCard.tsx:126]: The previous link was for retirement, is the change intentional? If it co, then it's basically the same as `learnMoreButton`

File: govtool/frontend/src/components/organisms/DashboardCards/DirectVoterDashboardCard.tsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ export const DirectVoterDashboardCard = ({
             dataTestId: "learn-more-button",
             onClick: () =>
               openInNewTab(
-                "https://docs.gov.tools/how-to-use-the-govtool/using-govtool/dreps/retire-as-a-drep",
+                "https://docs.gov.tools/using-govtool/govtool-functions/dreps/register-as-a-drep",
               ),
           },
```
Comment (jdyczka) [inline govtool/frontend/src/components/organisms/DashboardCards/DirectVoterDashboardCard.tsx:103]: Wouldn't https://docs.gov.tools/using-govtool/govtool-functions/direct-voting be a better suited link?

File: govtool/frontend/src/components/organisms/HomeCards.tsx [hunk 3]
```diff
@@ -34,17 +34,17 @@ export const HomeCards = () => {
...
   const onClickLearnMoreAboutDirectVoterRegistration = () =>
     openInNewTab(
-      "https://docs.gov.tools/how-to-use-the-govtool/using-govtool/direct-voting",
+      "https://docs.gov.tools/using-govtool/govtool-functions/direct-voting",
     );
 
   const onClickLearnMoreAboutProposingGovAction = () =>
     openInNewTab(
-      "https://docs.gov.tools/how-to-use-the-govtool/using-govtool/governance-actions/propose-a-governance-action",
+      "https://docs.gov.tools/using-govtool/govtool-functions/governance-actions/view-governance-actions/propose-a-governance-action",
```
Comment (jdyczka) [inline govtool/frontend/src/components/organisms/HomeCards.tsx:47]: I believe the correct link is https://docs.gov.tools/using-govtool/govtool-functions/governance-actions/propose-a-governance-action

Comment (MSzalowski) [reply govtool/frontend/src/components/organisms/HomeCards.tsx:47]: Good Catch!

File: govtool/frontend/src/components/organisms/DashboardCards/DRepDashboardCard.tsx (hunk 1 above)
Comment (MSzalowski) [reply govtool/frontend/src/components/organisms/DashboardCards/DRepDashboardCard.tsx:126]: It should be retire - thanks for spotting this! :saluting_face:

**Review by jdyczka — APPROVED** [review]

**Pull request merged**